src/platform/common/utils/multiStepInput.ts

```diff
@@ -41,7 +42,7 @@ export interface IQuickPickParameters<T extends QuickPickItem> {
     acceptFilterBoxTextAsSelection?: boolean;
     startBusy?: boolean;
     stopBusy?: Event<void>;
-    validate?(value: string): Promise<string | undefined>;
+    validate?(selection: T | QuickPick<T>): Promise<string | undefined>;
```

Comment:
I think passing a string was not the best approach with quick pick items, as the label doesn't always contain all of the information. Hence changed the code to pass the selected item instead of the label.